pages/learn/develop/multicontainer.md

```diff
@@ -113,7 +113,60 @@ In addition to the settings above, there are some {{ $names.company.lower }} spe
 
 {{> "general/labels" }}
 
-[docker-compose]:https://docs.docker.com/compose/overview/
+### Container requirements
+
+**Note** Container requirements are available when using balenaCLI >= 21.1.0
+
+An additional set of labels allows to ensure the compatibility of a device when it comes to run a service. For example, before updating to a new release, it may be desirable to ensure that the device is running a specific version of [Supervisor][supervisor] or has a specific version of the [NVIDIA Tegra Linux Driver Package][l4t] (L4T).
```

Comment:
This sentence reads a bit weirdly. I suggest something like:

"An additional set of labels ensures device compatibility for running a service. ..."